spec/lib/govuk_healthcheck/sidekiq_redis_spec.rb

```diff
@@ -3,35 +3,68 @@
 require_relative "shared_interface"
 
 RSpec.describe GovukHealthcheck::SidekiqRedis do
-  let(:redis_info) { double(:redis_info) }
-  let(:sidekiq) { double(:sidekiq, redis_info:) }
   before { stub_const("Sidekiq", sidekiq) }
 
-  context "when the database is connected" do
+  context "when the Sidekiq version is >= 7.0.0" do
```

Comment:
I think this (and the alternate version) should probably explicitly reference `default_configuration` rather than the Sidekiq version - as the test doesn't actually reference that and you have to implicitly understand that setting up default_configuration alludes to it being Sidekiq 7 or not.

Reply:
cool have updated it and added a comment to the healthcheck above the conditional explaining the issue.